h2o-py/h2o/frame.py

```diff
@@ -2706,7 +2706,6 @@ def split_frame(self, ratios=None, destination_frames=None, seed=None):
 
         splits = []
         tmp_runif = self.runif(seed)
-        tmp_runif.frame_id = "%s_splitter" % _py_tmp_key(h2o.connection().session_id)
```

Comment:
should I revert this one? or do we leave this change as is?

Reply:
I think we can keep it in the PR, I see no point giving something a pretty name if we are explicitly disposing later